src/runner/test-run/index.js

```diff
@@ -171,6 +185,18 @@ ServiceMessages[COMMAND.nativeDialogsInfoSet] = function (msg) {
     }
 };
 
-ServiceMessages[COMMAND.takeScreenshot] = function () {
-    //TODO:
+ServiceMessages[COMMAND.takeScreenshot] = async function (msg) {
+    if (!await exists(this.screenshotPath))
```

Comment:
We don't need this custom code here (and custom utilities), because we already have external module for this purposes: https://github.com/superroma/testcafe-phoenix/blob/master/src/cli/argument-parser.js#L15